resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.pl

```diff
@@ -388,126 +385,59 @@
 }
 
 #######################################
-### Annovar annotations
-#
-`cat $snvsGT_germlineRare_oVCF | perl \$TOOL_VCF_TO_ANNOVAR > $snvsGT_germlineRare_oVCF.forAnnovar.bed`;
-
-`\${ANNOVAR_BINARY} --buildver=\${ANNOVAR_BUILDVER} \${ANNOVAR_DBTYPE} $snvsGT_germlineRare_oVCF.forAnnovar.bed \${ANNOVAR_DBPATH}`;
-
-`perl \${TOOL_PROCESS_ANNOVAR} $snvsGT_germlineRare_oVCF.forAnnovar.bed.variant_function $snvsGT_germlineRare_oVCF.forAnnovar.bed.exonic_variant_function > $snvsGT_germlineRare_oVCF.forAnnovar.temp`;
-
-`perl \${TOOL_NEW_COLS_TO_VCF} -vcfFile=$snvsGT_germlineRare_oVCF --newColFile=$snvsGT_germlineRare_oVCF.forAnnovar.temp --newColHeader=ANNOVAR_FUNCTION,GENE,EXONIC_CLASSIFICATION,ANNOVAR_TRANSCRIPTS --reportColumns=3,4,5,6 --bChrPosEnd=0,1,2 > $snvsGT_germlineRare_oVCF_annovar`;
+### Counting the somatic numbers 
+open(my $SOMATIC_FH, "<$snvsGT_somatic")
+    || die "Can't open the file $snvsGT_somatic\n";
 
-## Separating rare germline and rescued somatic variants
-open(RG, ">$snvsGT_germlineRare_oVCF_annovar_rG") || die "$snvsGT_germlineRare_oVCF_annovar_rG can't be open for writing. $!";
-open(SR, ">$snvsGT_germlineRare_oVCF_annovar_sR") || die "$snvsGT_germlineRare_oVCF_annovar_sR can't be open for writing. $!";
-open(GRA, "<$snvsGT_germlineRare_oVCF_annovar") || die "can't open $snvsGT_germlineRare_oVCF_annovar $!";
-
-while(<GRA>){
-  my $tmp_GRA = $_;
-  chomp $tmp_GRA;
-  if($tmp_GRA=~/^#/) {
-    print RG "$tmp_GRA\n";
-    print SR "$tmp_GRA\n";
-  }
-  elsif($tmp_GRA=~/Germline|SomaticControlRare/){
-    print RG "$tmp_GRA\n";
-  }
-  elsif($tmp_GRA=~/Somatic_Rescue/){
-    print SR "$tmp_GRA\n";
-  }
-}
-close RG;
-close SR;
-close GRA;
-#######################################
-## Running Bias Filters
-
-my $runBiasScript_command = join("", "python '$biasScript' '$snvsGT_somatic' '$tumorBAM' '$ref' '$snvsGT_somaticRareBiasFile'",
-  " --tempFolder $analysisBasePath",
-  " --maxOpRatioPcr=0.34",
-  " --maxOpRatioSeq=0.34",
-  " --maxOpReadsPcrWeak=2",
-  " --maxOpReadsPcrStrong=2");
+while(!eof($SOMATIC_FH)) {
+  my $line = readline($SOMATIC_FH)
+    || die("Error reading from zcatted '$snvsGT_somatic': $!");
+  chomp $line;
   
-my $runBiasScript = system($runBiasScript_command);
-
-if($runBiasScript != 0) {
-  die "Error while running $biasScript in swapChecker: exit code = $runBiasScript\n";
-}
-
-
-### Counting The Numbers 
-open(SOM_RareBias, "<$snvsGT_somaticRareBiasFile")
-    || die "Can't open the file $snvsGT_somaticRareBiasFile\n";
-
-while(<SOM_RareBias>) {
-  chomp;
-  if($_!~/^#/) {
-    if($_=~/Tumor_Somatic_Common/ && $_!~/bPcr|bSeq/) {
+  if($line!~/^#/) {
+    if($line=~/Tumor_Somatic_Common/) {
       $json{'somaticSmallVarsInTumorCommonInGnomad'}++;
     }
-    elsif($_=~/Tumor_Somatic/ && $_=~/bPcr|bSeq/) {
-      $json{'somaticSmallVarsInTumorInBias'}++;
-    }
-    elsif($_=~/Tumor_Somatic_Rare/) {
+    elsif($line=~/Tumor_Somatic_Rare/) {
```

Comment:
Does the `/Tumor_Somatic/` case not occur? Or is this supposed to be an API-breaking change? But I see that this case prevented the `/Tumor_Somatic_Rare/` case to ever be detected.

Reply:
`/Tumor_Somati/` will be classified into `/Tumor_Somatic_Common/` or `/Tumor_Somatic_Rare/`